docs/integrations/dazz.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Dazz Integration
+category: 62cdf6722c2c1602a4b69643
+hidden: false
+---
+# Dazz Integration
+
+## Overview
+Phylum can be connected with Dazz to diaplay Phylum findings in the Dazz application.
+
+## Getting Started
+Current Dazz users can follow the installation instructions found on [Dazz](https://app.dazz.io/docs/en/phylum.html).
```

Comment:
This link appears to be behind their app's authentication...which is fine...but it can't be checked for accuracy at this time. Are we sure it is correct? Would it help to make that it bit more clear...maybe something like "Authenticated Dazz users can..." ?

Reply:
It is accurate today, but of course we don't control that. I don't care either way about using more words to describe the authentication requirements as they should get sent to a login page if not authenticated. Whatever you like is fine with me.

Reply:
Nah...good enough for now...lets wrap this one up.